src/three-components/Model.ts

```diff
@@ -108,18 +117,19 @@ export default class Model extends Object3D {
    *
    * @param {THREE.Object3D}
    */
-  setObject(model) {
+  setObject(model: Object3D) {
     this.clear();
     this.modelContainer.add(model);
     this.updateBoundingBox();
     this.dispatchEvent({type: 'model-load'});
   }
 
   /**
-   * @param {String} url
+   * @param {String?} url
```

Comment:
Again, should we have these decorations anymore?

Reply:
These are a bit redundant with TS